shc/web/interface.py

```diff
@@ -291,11 +317,35 @@ async def _page_handler(self, request: aiohttp.web.Request) -> aiohttp.web.Respo
 
         html_title = self.title_formatter(page.title)
         template = jinja_env.get_template('page.htm')
-        body = await template.render_async(title=page.title, segments=page.segments, menu=self.ui_menu_entries,
-                                           root_url=self.root_url, js_files=self._js_files, css_files=self._css_files,
-                                           server_token=id(self), html_title=html_title)
+        self._mark_active_menu_items(request.path)
```

Comment:
You should pass the page name (`page.name`) here instead of the full path from the request.

Reply:
You are so right.  Thanks 👍